examples/alphaTestExamples/init.md

```diff
@@ -0,0 +1,156 @@
+[kind]: https://github.com/kubernetes-sigs/kind
+
+# Demo: Init Command
+
+This demo shows how the kapply init command works.
+
+First define a place to work:
+
+<!-- @makeWorkplace @testE2EAgainstLatestRelease -->
+```
+DEMO_HOME=$(mktemp -d)
+```
+
+Alternatively, use
+
+> ```
+> DEMO_HOME=~/hello
```

Comment:
nit: Let's use `DEMO_HOME=~/demo`. By the `hello` folder, I thought the example was about helloworld.

Reply:
Fixed.